landingai/pipeline/image_source.py

```diff
@@ -217,7 +218,7 @@ class NetworkedCamera(BaseModel):
 
     def __init__(
         self,
-        stream_url: str,
+        stream_url: Union[str, int],
```

Comment:
remember to update docs about this new type.

Reply:
Good catch. Done! Thanks!